### PR TITLE
[Feature][Connector-V2][JDBC] Add presto connector

### PR DESCRIPTION
**Pull request by qianmoQ**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Review by hailin0 — COMMENTED** [review]
Should class names use `Trino` as a prefix?

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/pom.xml [hunk 1]
```diff
@@ -108,6 +114,12 @@
             <artifactId>terajdbc4</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/pom.xml:117]: Add into `connector-jdbc` dependencyManagement

https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-connectors-v2/connector-jdbc/pom.xml#L47

**Comment by qianmoQ** [comment]
> Should class names use `Trino` as a prefix?

No need, the core is based on the implementation of `io.presto`, only to do the jdbc conversion of `io.trino`

**Review by hailin0 — COMMENTED** [review]
add Drivers into dist module

https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml#L429
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/src/main/assembly/assembly-bin-ci.xml#L160

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/pom.xml (hunk 1 above)
Comment (qianmoQ) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/pom.xml:117]: Fixed

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+@Slf4j
+public class JdbcPrestoIT extends TestSuiteBase implements TestResource {
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:48]: Add `JdbcTrinoIT` and `JdbcPrestoIT` testing trino & presto docker images

Comment (qianmoQ) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:48]: trinodb/trino images are added to the test unit

**Comment by Hisoka-X** [comment]
Hi ,please fix ci first, thanks.

**Comment by qianmoQ** [comment]
> Hi ,please fix ci first, thanks.

please check it

**Comment by Hisoka-X** [comment]
Seem like the jdbc e2e failed.

**Comment by qianmoQ** [comment]
> Seem like the jdbc e2e failed.

Fixed， please merge

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        if (dbServer != null) {
+            dbServer.close();
+        }
+    }
+
+    @TestTemplate
+    @DisplayName("JDBC-Presto end to end test")
+    public void testJdbcPresto(TestContainer container) {
+        assertHasData();
+        log.info(container.toString());
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:102]: check read rows & fields form trino or presto

reference
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-fake-e2e/src/test/resources

Comment (hailin0) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:102]: @qianmoQ

**Comment by sanjaychitransh** [comment]
when will Presto connector   available?, i need to test for my project ,can you please help me

**Comment by sanjaychitransh** [comment]
@qianmoQ  i tested but connection with presto is failed,please help to close the PR

**Comment by sanjaychitransh** [comment]
is the code merged? may i know the update? this PR is closed

**Pull request closed without merging**